src/Model/Resolver/Menu.php

```diff
@@ -28,15 +25,12 @@
  */
 class Menu implements ResolverInterface
 {
-    /**
-     * @var ValueFactory
-     */
-    private $valueFactory;
+    const CATEGORY_ID_KEY = 'category_id';
```

Comment:
In latest PHP versions there are scoped constants, but usually these are public, however we still must declare the scope: **public const CATEGORY_ID_KEY=...**